paasta_tools/kubernetes_tools.py

```diff
@@ -2719,6 +2719,17 @@ def list_matching_deployments(
     )
 
 
+def list_matching_deployments_in_all_namespaces(
+    service: str,
+    instance: str,
+    kube_client: KubeClient,
+) -> Sequence[KubeDeployment]:
```

Comment:
should this be 
```suggestion
) -> Set[KubeDeployment]:
```
or are we sticking with the more abstract type to be consistent with the rest of this file?

Reply:
`list_deployments_in_all_namespaces` returns a Sequence so I just stuck with that. I could re-type `list_deployments_in_all_namespaces` to return a `List[KubeDeployment]`